### ISSUE TITLE
NPC quest issue

### ISSUE THREAD

**Issue by JaRy66** [YggdrasilWotLK/Yggdrasil-WoW#145]
**Title:** NPC quest issue (Vaelen the flayed

**Issue:** To explain shortly, in icecrown, as you do the quests to retake shadow Vault, you are sent to north Jotunheim, where in cave you find chained **Vaelen the flayed**, and he gives you series of quests, that can free him, and return him to the Shadow Vault.
However there are total 3 quests the NPC offers, 2 in his chained form, and one in his Shadow Vault form. **BUT** currently his chained form is offering all 3 quests, making it weird from the quest description side.

The npc's
https://www.wowhead.com/wotlk/npc=30056/vaelen-the-flayed (chained)
https://www.wowhead.com/wotlk/npc=30218/vaelen-the-flayed (shadow vault) 

There even is quest referencing him coming back to the vault:
https://www.wowhead.com/wotlk/quest=13085/vaelen-has-returned

**TL:DR***
The chained version of the NPC is giving player a quest, that the "free" version of NPC should give away

**Proof:** 
![WoWScrnShot_120224_002943](https://github.com/user-attachments/assets/1e082013-9374-40a8-a652-79a7ed68d2a0)

**Comment by JaRy66** [comment]
Update: The chained version also help the player fight, he shouldn't do that. He's chained

**Comment by YggdrasilWotLK** [comment]
@Lortz92 wanna change which NPCs the quests are connected to?

**Comment by YggdrasilWotLK** [comment]
Pacificity corrected by 

```
-- Pacifies chained Vaelen
UPDATE creature_template SET unit_flags = unit_flags | 131072 WHERE entry = 30056;
```

**Comment by YggdrasilWotLK** [comment]
I'm unable to reproduce the triple quest issue; After completing quest ID 12939, I'm getting offered the following quest:

1. Get the Key.  After completion, I get offered:

2. Let the Baron Know. This takes me to the Baron in Shadow Vault. After completing "Let the Baron Know", I can no longer see chained Vaelen. (His visibility is controlled by spell ID 27622.)

Moreover, chained Vaelen is only set to offer two quests in the database, with the unchained Vaelen offering the third.

![Image](https://github.com/user-attachments/assets/7e3feb49-b565-4362-a2d0-7e2f7f8732fe)
![Image](https://github.com/user-attachments/assets/6790fdc5-9e04-4893-9f9d-28239bbe8d0f)

This seems fixed since the report. Will reopen if new reports appear.
